app/javascript/packs/application.js

```diff
@@ -7,6 +7,13 @@ import Rails from "@rails/ujs"
 import Turbolinks from "turbolinks"
 import * as ActiveStorage from "@rails/activestorage"
 import "channels"
+import 'bootstrap'
+require("@rails/ujs").start()
+require("turbolinks").start()
+require("@rails/activestorage").start()
```

Comment:
You don't need to require these libs again since those were already imported using ES6 modules. Feel free to get rid of them.